targets/TARGET_RDA/TARGET_UNO_91H/lib/README.md

```diff
@@ -0,0 +1,4 @@
+libhal files in the subfolders are generated with toolchains:
+* **Arm Compiler 5** - version 5.06u1
+* **GNU Arm Embedded** - version 4.9.3
```

Comment:
Can this be updated to gcc 6 ?

What actually is in this archive? SDK drivers?

Reply:
Archive is mainly about sleep strategy.